templates/gallery/gallery-map.html

```diff
@@ -0,0 +1,318 @@
+              "visibility": "off"
+            }
+          ]
+        },
+        {
+          "featureType": "road.local",
+          "elementType": "labels.text.fill",
+          "stylers": [
+            {
+              "color": "#9e9e9e"
+            }
+          ]
+        },
+        {
+          "featureType": "transit",
+          "stylers": [
+            {
+              "visibility": "off"
+            }
+          ]
+        },
+        {
+          "featureType": "transit.line",
+          "elementType": "geometry",
+          "stylers": [
+            {
+              "color": "#e5e5e5"
+            }
+          ]
+        },
+        {
+          "featureType": "transit.station",
+          "elementType": "geometry",
+          "stylers": [
+            {
+              "color": "#eeeeee"
+            }
+          ]
+        },
+        {
+          "featureType": "water",
+          "elementType": "geometry",
+          "stylers": [
+            {
+              "color": "#c9c9c9"
+            }
+          ]
+        },
+        {
+          "featureType": "water",
+          "elementType": "labels.text.fill",
+          "stylers": [
+            {
+              "color": "#9e9e9e"
+            }
+          ]
+        }
+      ]
+    </script>
+    <script>
+      function initMap() {
+        var query = window.location.search.replace(/^\?/, '').split('&')
+          .reduce(function (result, v) {
+            var parts = v.split('=');
+            var key = decodeURIComponent(parts[0]);
+            var val = decodeURIComponent(parts[1]);
+            if (result.hasOwnProperty(key)) {
+              if (Array.isArray(result[key])) {
+                result[key].push(val);
+              } else {
+                result[key] = [result[key], val];
+              }
+            } else {
+              result[key] = val;
+            }
+            return result;
+          }, {});
+
+        var rawMarkerPos = query.marker.split(',');
+        var markerPos = {lat: parseFloat(rawMarkerPos[0]), lng: parseFloat(rawMarkerPos[1])};
+        var map = new google.maps.Map(document.getElementById('map'), {
+          zoom: 14,
+          center: markerPos,
+          scrollwheel: false,
+          streetViewControl: false,
+          backgroundColor: 'none',
+          styles: JSON.parse(document.getElementById('style').innerHTML)
+        });
+
+        map.panBy(-window.innerWidth * 0.1, -window.innerHeight * 0.15);
+
+        var marker = new google.maps.Marker({
+          position: markerPos,
+          map: map,
+          icon: {
+            url: 'data:image/svg+xml;charset=UTF-8;base64,' + btoa(document.getElementById('marker').innerHTML),
+            size: new google.maps.Size(80, 80),
+            origin: new google.maps.Point(0, 0),
+            anchor: new google.maps.Point(40, 40)
+          }
+        });
+      }
+    </script>
+    <!-- NOTE: Maps API key is from https://ampbyexample.com/components/amp-iframe/ -->
```

Comment:
Open a bug to create a maps API for ampstart
Add a todo for me here link to the bug

Reply:
Do you want the bug opened against this repo or that main amp repo?

Reply:
This Repo